packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -352,7 +357,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         aria-expanded={ this._isExpanded }
         aria-pressed={ this.props.checked }
         aria-describedby={ buttonProps.ariaDescription }
-        className={ css(disabled ? styles.splitButtonContainerDisabled : styles.splitButtonContainer) }
+        className={ css(disabled ? splitButtonStyles.splitButtonContainerDisabled : splitButtonStyles.splitButtonContainer) }
```

Comment:
Shouldn't this get the class names from the given styles that are passed into props?